src/components/Match/matchColumns.jsx

```diff
@@ -657,7 +657,7 @@ export default (strings) => {
                 })
                 .map((purchase) => {
                   if (items[purchase.key] && (showConsumables || items[purchase.key].qual !== 'consumable')) {
-                    return inflictorWithValue(purchase.key, formatSeconds(purchase.time));
+                    return inflictorWithValue(purchase.key, purchase, 'purchase');
```

Comment:
instead of changing the type of the second argument can we just add a new additional one for the charges overlay?

Reply:
I think I've made the changes you've requested, however I'm pretty sure I've screwed up the commit history a bit. Sorry about this I'm new to prs.